src/main/java/org/checkerframework/specimin/UnsolvedMethod.java

```diff
@@ -0,0 +1,36 @@
+package org.checkerframework.specimin;
+
+/**
+ * An UnsolvedMethod instance is a representation of a method that can not be solved by
+ * SymbolSolver. The reason is that the class file of that method is not in the root directory.
+ */
+public class UnsolvedMethod {
+  /** The name of the method */
+  public String name;
```

Comment:
Same comment about these fields being final. We might want to enable a linter like [ErrorProne](https://errorprone.info/) so that you're alerted when you're not following a best practice like this one.